service-jehlomat/src/test/resources/application.conf

```diff
@@ -0,0 +1,22 @@
+ktor {
+    deployment {
+        port = 8089
+    }
+
+    databaseConfiguration {
+        host = "localhost"
+        port = "5432"
+        database = "jehlomat"
+        user = "jehlomat"
+        password = "jehlomat"
+    }
+
+    application {
+        modules = [ main.ApplicationKt.module ]
+    }
+}
+
+mailjet {
+    publicKey = "cbb81504a06f2fd735db577f09666b7f"
```

Comment:
I would say these secret values shouldn't be committed. Moreover, we don't use application context for database and mailjet anymore.

Reply:
Removed but put to readme. 

We decided to public api keys, see our communication from yesterday:
```
[Juraj Kisztner]
nech je tak prosim
[8:21]
pregenerujeme je kdyz dokoncime jehlomat
[8:21]
co jsem se dival tak zatim je zadny robot nepouzil :slightly_smiling_face:
[8:22]
```

Reply:
https://github.com/cesko-digital/jehlomat/pull/163/commits/1a5d6f732238dd2f8c14ee0dca34cc252b3157c2